sc/shared/src/test/scala/sigmastate/ScriptVersionSwitchSpecification.scala

```diff
@@ -266,44 +266,45 @@ class ScriptVersionSwitchSpecification extends SigmaDslTesting {
 
   /** Rule#| BlockVer | Block Type| Script Version | Release | Validation Action
     * -----|----------|-----------|----------------|---------|--------
-    * 19   | 4        | candidate | Script v3      | v5.0    | skip-accept (rely on majority)
-    * 20   | 4        | mined     | Script v3      | v5.0    | skip-accept (rely on majority)
+    * 19   | 4        | candidate | Script v3      | v6.0    | R6.0-JIT-verify
+    * 20   | 4        | mined     | Script v3      | v6.0    | R6.0-JIT-verify
```

Comment:
this doesn't look right. why Validation Action changed?
The whole set of rules is designed for two versions V and V+1 and describes switch from current version to the next version.
The semantics of each rule should remain the same, but versions should move from v to v+1.

Reply:
Changed to Block V5 with Script V4 activated